vars/piperPipelineStageRelease.groovy

```diff
@@ -84,6 +84,8 @@ void call(Map parameters = [:]) {
 
             if (config.kubernetesDeploy){
                 durationMeasure(script: script, measurementName: 'deploy_release_kubernetes_duration') {
+                    helmExecute script: script, helmCommand: 'dependency', dependency: 'update'
+                    stash name: 'charts', includes: "${config.chartPath}/charts/*.tgz", allowEmpty: true
```

Comment:
Would be better to provide the `charts` stash also to `kubernetesDeploy`.
But this replaces other stashes configured somewhere else in the configuration. This means: when we e.g. configure the `source` stash somewhere else, this is lost. Would be better in this case when the stashes are added instead of replaced ...

Reply:
Not sure if `/charts/*.tgz"` above needs to be configurable.

Reply:
Maybe we should have a flag which controls if `helmExecute` for dependency update is executed.